src/tests/unit/platform/test_users.py

```diff
@@ -269,6 +255,27 @@ def test_get_grant(cbcsdk_mock):
     assert grant.roles == ["psc:role::SECOPS_ROLE_MANAGER", "psc:role:test:APP_SERVICE_ROLE"]
 
 
+def test_delete_user_without_grant(cbcsdk_mock):
```

Comment:
I think those tests will always pass since there is no assertion or anything that checks if the user is actually deleted.

Reply:
_Any_ exception will be a "fail" on the test. There shouldn't be a need to check for a specific one.

Reply:
This has been done. Take the new delete() code out now and the two tests will both fail with the assert message, "Grant should have been checked for prior to deleting user".